packages/bpk-component-button/src/BpkButtonV2/BpkButton.tsx

```diff
@@ -23,7 +23,8 @@ import COMMON_STYLES from './BpkButton.module.scss';
 
 const getCommonClassName = cssModules(COMMON_STYLES);
 
-const BpkButtonV2 = ({
+// eslint-disable-next-line import/prefer-default-export
+export const BpkButtonV2 = ({
```

Comment:
for some reason the problem with react docgen was the way the button was exported.. I changed it to a named export because that's what we do for modalv2 and shouldn't matter as in the end the export to consumers stays the same